tests/Test/ConfigMysqlTest.php

```diff
@@ -68,6 +71,8 @@ public function setUp(): void
 
         $this->userRepository = $this->getTestContainer()->get('doctrine')
             ->getRepository('LiipAcme:User');
+
+        $this->databaseTool = $this->getTestContainer()->get(DatabaseToolCollection::class)->get();
```

Comment:
TODO: define `__invoke()` in `DatabaseToolCollection::class`, to get the fixtures loaded without having to call `->get()` here?

Reply:
I think between 

```php
$this->getTestContainer()->get(DatabaseToolCollection::class)->get()
```

and 

```
$this->getTestContainer()->get(DatabaseToolCollection::class)()
```

The first one is more obvious. If I understand correctly how `__invoke` should work.

Reply:
Oops, I thought `()` wasn't necessary. I agree that `get(DatabaseToolCollection::class)()` is neither nice visually nor useful. :+1: